src/styles/index.scss

```diff
@@ -1,3 +1,156 @@
-body {
+// Змінні
+$font-family: roboto, helvetica, sans-serif;
+$primary-color: #060b35;
+$secondary-color: #616070;
+$link-color: #00acdc;
+
+// Стилі для базових HTML елементів
+body,
+div,
+p,
+a {
   margin: 0;
+  box-sizing: border-box;
+  font-family: $font-family;
+}
+
+// Стилі для контейнера
+.container {
+  display: flex;
+  width: 200px;
+  height: 406px;
+
+  // Стилі для картки
+  .card {
+    padding: 32px 16px 16px;
+    border-radius: 5px;
+    border: 1px solid #f3f3f3;
+    background: #fff;
+
+    // Стилі для зображення в картці
+    .card__image {
+      display: block;
+      width: 160px;
+      height: 134px;
+      margin: 0 auto;
+    }
```

Comment:
checklist 1 - [STYLES] - Get used to style all elements using classes. And **don't increase selectors specificity** unless completely necessary.
_You've made style for each element inside container style sekector_